app/Components/AudioPlayer/Components/Arrow/Arrow.tsx

```diff
@@ -0,0 +1,11 @@
+import Image from "next/image";
+import styles from './Arrow.module.scss';
+import Link from 'next/link';
+
+const ArrowLink = () => (
+        <Link href='#' className={styles.arrow}>
```

Comment:
What is this link for?

Reply:
music page isnot ready for now it is not permanent